ios/AppsFlyerLib-Swift.h

```diff
@@ -0,0 +1,331 @@
+#if 0
+#elif defined(__arm64__) && __arm64__
```

Comment:
@al-af this removes support for us to compile our app with x86_64, is this intentional? this blocks us in CI given our setup still relies on Intel chip macs.